plugins/vst_base/CMakeLists.txt

```diff
@@ -24,40 +24,38 @@ SET(EXTERNALPROJECT_CMAKE_ARGS
 	"-DLMMS_SOURCE_DIR=${CMAKE_SOURCE_DIR}"
 	"-DLMMS_BINARY_DIR=${CMAKE_BINARY_DIR}"
 	"-DCMAKE_RUNTIME_OUTPUT_DIRECTORY=${CMAKE_RUNTIME_OUTPUT_DIRECTORY}"
-	"-DCMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG=${CMAKE_RUNTIME_OUTPUT_DIRECTORY}"
 	"-DCMAKE_MODULE_PATH=${CMAKE_MODULE_PATH}"
 	"-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
 )
 
+IF(MSVC)
+	LIST(APPEND EXTERNALPROJECT_CMAKE_ARGS "-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}")
```

Comment:
Since this is only used in the `ELSEIF(LMMS_BUILD_WIN64 AND MSVC) ` case, you can just move it down there, so that it reads from line 50:

```cmake
ExternalProject_Add(RemoteVstPlugin32
	"${EXTERNALPROJECT_ARGS}"
	CMAKE_GENERATOR ${GENERATOR}
	CMAKE_GENERATOR_TOOLSET ${CMAKE_GENERATOR_TOOLSET}
	CMAKE_ARGS
		"${EXTERNALPROJECT_CMAKE_ARGS}"
		-DCMAKE_TOOLCHAIN_FILE="${CMAKE_TOOLCHAIN_FILE}"
		"-DCMAKE_PREFIX_PATH=${QT_32_PREFIX}"
)
```